app/views/projects/show.html.erb

```diff
@@ -18,7 +18,8 @@
             param_data: params,
             event_names: @event_names,
             properties: @properties, 
-            possible_aggregation: @possible_aggregation
+            possible_aggregations: @possible_aggregations,
+            aggregation: @aggregation # ??? do we need this?, also in _selections.html.erb, or better @aggregation there?
```

Comment:
not sure if @aggregation is needed here in the locals

Reply:
It looks like it's needed, because you need to acces it` _selections.html.erb` no?